internal/scti/handlers.go

```diff
@@ -63,53 +62,38 @@ var (
 	// Metrics are all per-log (label "origin"), but may also be
 	// per-entrypoint (label "ep") or per-return-code (label "rc").
 	once             sync.Once
-	knownLogs        *prometheus.GaugeVec     // origin => value (always 1.0)
-	lastSCTIndex     *prometheus.GaugeVec     // origin => value
-	lastSCTTimestamp *prometheus.GaugeVec     // origin => value
-	reqsCounter      *prometheus.CounterVec   // origin, op => value
-	rspsCounter      *prometheus.CounterVec   // origin, op, code => value
-	rspLatency       *prometheus.HistogramVec // origin, op, code => value
+	knownLogs        metric.Int64Gauge       // origin => value (always 1.0)
+	lastSCTIndex     metric.Int64Gauge       // origin => value
+	lastSCTTimestamp metric.Int64Gauge       // origin => value
+	reqsCounter      metric.Int64Counter     // origin, op => value
+	rspsCounter      metric.Int64Counter     // origin, op, code => value
+	rspLatency       metric.Float64Histogram // origin, op, code => value
 )
 
 // setupMetrics initializes all the exported metrics.
 func setupMetrics() {
 	// TODO(phboneff): add metrics for deduplication and chain storage.
-	knownLogs = promauto.NewGaugeVec(
-		prometheus.GaugeOpts{
-			Name: "known_logs",
-			Help: "Set to 1 for known logs",
-		},
-		[]string{"origin"})
-	lastSCTTimestamp = promauto.NewGaugeVec(
-		prometheus.GaugeOpts{
-			Name: "last_sct_timestamp",
-			Help: "Time of last SCT in ms since epoch",
-		},
-		[]string{"origin"})
-	lastSCTIndex = promauto.NewGaugeVec(
-		prometheus.GaugeOpts{
-			Name: "last_sct_index",
-			Help: "Index of last SCT",
-		},
-		[]string{"origin"})
-	reqsCounter = promauto.NewCounterVec(
-		prometheus.CounterOpts{
-			Name: "http_reqs",
-			Help: "Number of requests",
-		},
-		[]string{"origin", "ep"})
-	rspsCounter = promauto.NewCounterVec(
-		prometheus.CounterOpts{
-			Name: "http_rsps",
-			Help: "Number of responses",
-		},
-		[]string{"origin", "op", "code"})
-	rspLatency = promauto.NewHistogramVec(
-		prometheus.HistogramOpts{
-			Name: "http_latency",
-			Help: "Latency of responses in seconds",
-		},
-		[]string{"origin", "op", "code"})
+	knownLogs = mustCreate(meter.Int64Gauge("tesseract.known_logs",
+		metric.WithDescription("Set to 1 for known logs")))
+
+	lastSCTTimestamp = mustCreate(meter.Int64Gauge("tesseract.last_sct.timestamp",
+		metric.WithDescription("Time of last SCT since epoch"),
+		metric.WithUnit("ms")))
```

Comment:
(To avoid ambiguity with my other comment, let's keep ms here since this is the unit in the RFC)